plugin.json

```diff
@@ -26,7 +24,7 @@
                 "key": "TeamChannel",
                 "display_name": "Channels to send notifications to:",
                 "type": "text",
-                "help_text": "The channels to send notifications to. Multiple channels can be specified. Format `teamname,channelname;teamname-2,channelname-2`. If the specified channels do not exist, the plugin will create the channels for you.\n \n Note: Must be the team and channel handle used in the URL. For example, in the following URL, set the value to `myteam,mychannel`: https://example.com/myteam/channels/mychannel.",
+                "help_text": "The channels to send notifications to. Multiple channels can be specified. Format 'teamname,channelname;teamname-2,channelname-2'. If the specified channels do not exist, the plugin will create the channels for you.\n \n Note: Must be the team and channel handle used in the URL. For example, in the following URL, set the value to 'myteam,mychannel': https://example.com/myteam/channels/mychannel.",
```

Comment:
Is it possible to escape the backtick with a backslash?

```suggestion
                "help_text": "The channels to send notifications to. Multiple channels can be specified. Format \`teamname,channelname;teamname-2,channelname-2\`. If the specified channels do not exist, the plugin will create the channels for you.\n \n Note: Must be the team and channel handle used in the URL. For example, in the following URL, set the value to 'myteam,mychannel': https://example.com/myteam/channels/mychannel.",
```

Reply:
@mickmister I don't think so as it is an invalid escape character and the VS code is showing the same error 
```
Invalid escape character in string.json(261)
```

Reply:
@mickmister  I have tried but the problem here is that when we are creating the manifest file it contains the manifestStr string which basically contains the plugin.json settings that are created inside the backticks as it is a multi-line string and if we try to use backticks in any of the plugin.json settings that will be generated in that file as well and we cannot use backticks inside backticks and this will error out. 
![Screenshot from 2024-03-21 14-33-53](https://github.com/mattermost/mattermost-plugin-aws-SNS/assets/90389917/8488e1c9-491f-41b4-82a1-15175c07691e)